hw16/index.js

```diff
@@ -0,0 +1,151 @@
+'use strict'
+
+/*
+var admin;
+let userName;
+
+userName='Джон';
+admin=userName;
+
+console.log(admin);
+
+alert(admin)
+
+ourPlanet="Наша Планета"
+
+currentUserOfWebsite
+
+*/
+
+//TASK 1
+
+/*
+let x = 20;
+let y = 58;
+let z = 42;
+let plusResult = x + y + z
+
+console.log(plusResult)
+
+*/
+
+//TASK 2
+
+/*
+let a = 1;
+
+let minute = a * 60;
+let hours = minute * 60;
+let day = hours * 24;
+let year = day * 365;
```

Comment:
Лучше реализовать таким образом:
```suggestion
const SECONDS_IN_MINUTE = 60;
const MINUTES_IN_HOUR = 60;
const HOURS_IN_DAY = 24;
const DAYS_IN_YEAR = 365;

const myAgeInSeconds = 27 * DAYS_IN_YEAR * HOURS_IN_DAY * MINUTES_IN_HOUR * SECONDS_IN_MINUTE;
```